src/vs/workbench/contrib/welcome/page/browser/welcomePage.ts

```diff
@@ -130,8 +132,17 @@ function isWelcomePageEnabled(configurationService: IConfigurationService, conte
 			return welcomeEnabled.value;
 		}
 	}
-	// {{SQL CARBON EDIT}} - add welcomePageWithTour
-	return startupEditor.value === 'welcomePageWithTour' || startupEditor.value === 'welcomePage' || startupEditor.value === 'readme' || startupEditor.value === 'welcomePageInEmptyWorkbench' && contextService.getWorkbenchState() === WorkbenchState.EMPTY;
+	return startupEditor.value === 'welcomePage' || startupEditor.value === 'readme' || startupEditor.value === 'welcomePageInEmptyWorkbench' && contextService.getWorkbenchState() === WorkbenchState.EMPTY;
+}
+
+
+// {{SQL CARBON EDIT}} - add isInitialSetupWizardEnabled
+function isInitialSetupWizardEnabled(configurationService: IConfigurationService): boolean {
```

Comment:
Why do you need to make the changes here? You're already doing these checks in the SQL welcome page so it seems unnecessary to duplicate that here